.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  root: true,
+  extends: ["eslint:recommended"],
+  parserOptions: {
+    ecmaVersion: "latest",
+    sourceType: "module",
```

Comment:
this should be done in `overrides` only for `.mjs` files, not globally.

Reply:
done